Code_source/Compiled/control/keyboard.c

```diff
@@ -340,6 +340,7 @@ static void keyboard_vis(t_gobj *z, t_glist *glist, int vis){
 // ------------------------- Methods ------------------------------
 void keyboard_float(t_keyboard *x, t_floatarg f){
     int note = (int)f;
+    if(note < 0 || note >= 255) return;
```

Comment:
maybe i'm missing sth - but returning for `note >= 255` doesn't reflect the commit message, which is saying "above 255"?